barretenberg/cpp/src/barretenberg/transcript/transcript.hpp

```diff
@@ -450,6 +450,15 @@ static bb::StdlibProof<Builder> convert_proof_to_witness(Builder* builder, const
     return result;
 }
 
+template <typename Builder> static bb::HonkProof convert_stdlib_proof_to_native(const StdlibProof<Builder>& proof)
```

Comment:
We should try to use constructors whenever possible to represent the construction of objects. Here is a pattern we can use to do that in this case: https://godbolt.org/z/odT7MsG7r.